### PR TITLE
Adding lensed point sources light profile

### PR DESCRIPTION
**Pull request by LyneVdV**
I don't know if it's the best way to do it but I basically created a NonLinearParameterSet (as a LinearParameterSet already exists) to enable a "ra_list" and "dec_list" for lensed point sources positions. The amplitudes "amps" are a LinearParameterSet.

### CODE REVIEW

File: python/lensmodelapi/api/parameter.py [hunk 1]
```diff
@@ -144,6 +145,12 @@ class LinearParameterSet(ParameterSet):
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
+        
+class NonLinearParameterSet(ParameterSet):
+    """Typically for pixelated profiles"""
```
Comment (aymgal) [inline python/lensmodelapi/api/parameter.py:150]: I guess this line has not been updated for the new class.

**Review by aymgal — CHANGES_REQUESTED** [review]
Looks good to me, thanks a lot Lyne ! Just this little docstring line to be updated.

File: python/lensmodelapi/api/parameter.py (hunk 1 above)
Comment (LyneVdV) [reply python/lensmodelapi/api/parameter.py]: yes, good catch, thanks !

**Comment by LyneVdV** [comment]
Should be ok now !

**Review by aymgal — APPROVED** [review]

**Pull request merged**